### ISSUE TITLE
Questions around contributing & docs

### ISSUE THREAD

**Issue by cspalding** [featurehub-io/featurehub-ruby-sdk#4]
Hey there! This SDK is awesome. My team is looking into using Featurehub within a Rails application, and using this SDK appears to be preferable to us compared to building a client from the [OpenAPI Spec](https://docs.featurehub.io/featurehub/latest/sdks-development.html#_openapi) (which is also great to have). I do have a couple of questions about the project and I'm hoping this is the correct place to ask. If not, feel free to close the issue and, if you have time, point me in the right direction.

I see that this is a very fresh repo (first commit was around 18 days ago at the time of this writing). It appears that it went through a few days of rapid development but has slowed down a bit. Are y'all still working on updating the SDK at all, or is development complete for the foreseeable future?

If there is any ongoing development, do you have any public documentation around what features are still being added or changed? Similarly, I assume since this is open-source y'all would accept PR's from outside of the featurehub organization (within reason), but I wanted to verify that assumption.

Thanks in advance for any info you can provide! And thanks again for developing this SDK, it's really great.

**Comment by rvowles** [comment]
Hey @cspalding - yes, fresh repo, i wrote the ruby version after i finished the python one, and then backtracked and changed some stuff in the python one based on changes in  the ruby one. I tried to make mostly feature parity with the c# version, the only thing I'm missing is analytics support and OpenTelemetry baggage support but I'm not sure quite how I'm going to approach that, I may make that a different gem.

I have finished development on it for the time being, but we are totally open to any 3rd party contributors, or even suggestions for making it better in any way. Docs are very gratefully received, as are examples (I had hoped to do a Rails example as well, as it is part of what I do in my day job).

So any contributions are welcome!
